### ISSUE TITLE
"Slide Background from left to right" does not respect "Preserve background aspect ratio"

### ISSUE THREAD

**Issue by Blade04208** [Keksuccino/FancyMenu#663]
**Describe the bug**
When having a wide image set to the background, having "Slide image from left to right" enabled, and "Preserve background aspect ratio" __disabled__ , Fancymenu will act as if Preserve background aspect ratio" is enabled.

**To Reproduce**
set a wide image to the background (i have provided my one in the screenshots section)
make sure "Slide image from left to right" is enabled
disable "Preserve background aspect ratio"
Save layout and exit
Image is distorted, not following the fact that "Preserve background aspect ratio" is disabled

**Expected behavior**
A non-distorted image sliding from left to right as the background

**Game Log**
Log is not required, fancymenu does not send any logs with this.

**Screenshots**
![image](https://github.com/Keksuccino/FancyMenu/assets/74104258/e9bbad0c-6d35-421d-84ce-8bdbec19139e)
![image](https://github.com/Keksuccino/FancyMenu/assets/74104258/52fb4040-fcb3-4e3e-9e3b-39ee68808027)
![image](https://github.com/Keksuccino/FancyMenu/assets/74104258/cddfc668-1f61-4e90-9378-46d30e24ea86)
## Expected behaviour:
![image](https://github.com/Keksuccino/FancyMenu/assets/74104258/a7058a36-3eb9-493f-b773-636b78f9450a)
### [Background image too big for github](https://im.ge/i/b5Jp0J)

**Basic Informations (please complete the following information):**
 - OS: Linux (EndeavourOS)
 - FancyMenu Version: 3.04
 - Forge/Fabric Version 0.15.3
 - Minecraft Version 1.20.4
 - Active Mods: see below 
```
  [✔] appleskin-fabric-mc1.20.3-2.5.1
  [✔] BetterF3-9.0.0-Fabric-1.20.4
  [✘] blur-3.2.0.jar (disabled)
  [✔] c2me-fabric-mc1.20.4-0.2.0+alpha.11.40
  [✔] capes-1.5.3+1.20.2-fabric
  [✔] chat_heads-0.10.31-fabric-1.20.3
  [✔] chatpatches-204.6.2
  [✔] cloth-config-13.0.121-fabric
  [✔] continuity-3.0.0-beta.4+1.20.2
  [✔] Controlify-1.8.1+1.20.4
  [✔] custom_hud-3.3.0+1.20.4
  [✔] dark-loading-screen-1.6.14
  [✘] DistantHorizons-2.0.1-a-1.20.2.jar (disabled)
  [✔] dynamic-fps-3.3.3+minecraft-1.20.0
  [✔] dynamiccrosshair-7.6.4+1.20.4-fabric
  [✔] enhancedblockentities-0.9.1+1.20.2
  [✔] entity_texture_features_fabric_1.20.4-5.2.2
  [✔] entityculling-fabric-1.6.3.1-mc1.20.4
  [✔] fabric-api-0.95.3+1.20.4
  [✔] fabric-language-kotlin-1.10.17+kotlin.1.9.22
  [✔] fancymenu_fabric_3.0.4_MC_1.20.4
  [✔] ferritecore-6.0.3-fabric
  [✔] ForgeConfigAPIPort-v20.4.3-1.20.4-Fabric
  [✔] Gamma-Utils-1.7.18-mc1.20.3
  [✔] held-item-info-1.5.0
  [✔] Iceberg-1.20.4-fabric-1.1.18
  [✔] ImmediatelyFast-Fabric-1.2.8+1.20.4
  [✔] indium-1.0.28+mc1.20.4
  [✔] InventoryProfilesNext-fabric-1.20.3-1.10.9
  [✔] Jade-1.20.4-fabric-13.2.1
  [✔] konkrete_fabric_1.8.0_MC_1.20.4
  [✔] lazydfu-0.1.3
  [✔] LegendaryTooltips-1.20.2-fabric-1.4.5
  [✔] libIPN-fabric-1.20.3-4.0.1
  [✔] lithium-fabric-mc1.20.4-0.12.1
  [✔] malilib-fabric-1.20.4-0.18.0
  [✔] melody_fabric_1.0.1_MC_1.20.1-1.20.4
  [✔] modernfix-fabric-5.12.1+mc1.20.4
  [✔] modmenu-9.0.0
  [✔] moreculling-1.20.4-0.22.1
  [✔] NoChatReports-FABRIC-1.20.4-v2.5.0
  [✔] noisium-1.0.2+mc1.20.x
  [✔] Pick-Block-Pro-1.7.24-mc1.20.4
  [✔] Prism-1.20.4-fabric-1.0.6
  [✔] raised-fabric-1.20.4-3.1.1
  [✔] rebind-all-the-keys-1.4.0+1.20.2
  [✔] reeses_sodium_options-1.7.0+mc1.20.2-build.97
  [✔] Resourcify (1.20.2+-fabric)-1.3.0
  [✔] rrls-4.0.1+mc1.20.4-fabric
  [✔] screenshot-to-clipboard-1.0.9-fabric
  [✔] show-me-your-skin-1.7.1+1.20.4
  [✔] sodium-extra-0.5.3+mc1.20.2-build.114
  [✔] sodium-fabric-mc1.20.3-0.5.5
  [✔] status-effect-bars-1.0.4
  [✔] stendhal-1.4.1-1.20.2
  [✔] suggestion-tweaker-1.20-1.5.1-fabric
  [✔] threadtweak-fabric-1.20.4-0.1.2
  [✔] toggle-scoreboard-1.2.0
  [✔] tweakeroo-fabric-1.20.4-0.19.1
  [✔] vivecraft-1.20.4-1.1.6-fabric
  [✔] vmp-fabric-mc1.20.4-0.2.0+beta.7.136-all
  [✔] waveycapes-fabric-1.4.2-mc1.20.4
  [✔] Xaeros_Minimap_23.9.7_Fabric_1.20.4
  [✔] XaerosWorldMap_1.37.7_Fabric_1.20.4
  [✔] yet-another-config-lib-fabric-3.3.2+1.20.4
  [✔] Zoomify-2.13.0
```
this is such a dumb and simple bug, but i really want it fixed.

**Comment by Keksuccino** [comment]
I think you confused what "Preserve Background Aspect Ratio" does.
**Enabling** it will show the image with the correct aspect ratio and **disabling** will **NOT**.

I will make it respect the "Preserve Aspect Ratio" option, but it will show with the correct aspect ratio when that option is **enabled**.

It should already show the correct aspect ratio if it's actually sliding. It only gets stretched when the image is not wide enough for the screen.

**Comment by Keksuccino** [comment]
Fixed/Added in FancyMenu v3.0.5!
